.github/workflows/prod_codesync.yml

```diff
@@ -0,0 +1,31 @@
+name: sync up to codecommit in prod
+
+on:
+  push:
+    tags-ignore:
+      - '*'
+    branches:
+      - '*'
+
+jobs:
+  prodsync:
+    runs-on: ubuntu-latest
+    permissions:
+      id-token: write
+      contents: read
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v3
+        with:
+          fetch-depth: 0
+      - name: Configure AWS Credentials
+        uses: aws-actions/configure-aws-credentials@v2
+        with:
+          role-to-assume: arn:aws:iam::840364872350:role/github-codesync-role
+          role-session-name: prodrolesession
+          aws-region: us-west-2
+      - name: Sync up to CodeCommit
+        uses: tsgit18/sync-up-to-codecommit-action@v1.0.6
```

Comment:
Do we need to watch out for updates of this action?

Reply:
Good point. Currently it's enough for us. If there is a significant change for it, then we can update to a new version. I think we won't need to update it very often.